src/workerd/api/r2-admin.h

```diff
@@ -23,6 +31,16 @@ class R2Admin: public jsg::Object {
   // `subrequestChannel` is what to pass to IoContext::getHttpClient() to get an HttpClient
   // representing this namespace.
 
+  R2Admin(FeatureFlags featureFlags,
```

Comment:
Just to avoid confusion about which constructor is used for what (i.e. created directly via binding vs internal support to be constructed from a sibling binding). It's the pattern established in the R2 bindings.